### PR TITLE
IBX-458: Document switching designs

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-458
| Versions      | 3.3, master

Note: links in this rely on https://github.com/ezsystems/developer-documentation/pull/1406, so will be broken until that PR is merged and I can rebase.

### CODE REVIEW

File: docs/guide/content_rendering/design_engine/add_new_design.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+``` yaml
+ezplatform:
+    siteaccess:
+        list: [site, campaign]
+        groups:
+            site_group: [site, campaign]
+        default_siteaccess: site
+```
+
+Adding the `campaign` SiteAccess to [`site_group`](../../multisite/multisite_configuration.md#siteaccess-groups) enables you to add common configuration for both SiteAccesses at the same time.
```
Comment (juskora) [inline docs/guide/content_rendering/design_engine/add_new_design.md:23]: Broken link for Multisite

File: docs/guide/content_rendering/design_engine/add_new_design.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+!!! tip
+
+    For details about configuring different site roots and matching SiteAccesses, see [Set up campaign SiteAccess](../../multisite/setup_campaign_siteaccess.md).
```
Comment (juskora) [inline docs/guide/content_rendering/design_engine/add_new_design.md:27]: Broken link

File: docs/guide/content_rendering/design_engine/add_new_design.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+## Create themes
+
+Next, configure a new `summersale` design this a theme also named `summersale`:
```
Comment (juskora) [inline docs/guide/content_rendering/design_engine/add_new_design.md:31]: I think some part of this sentence is missing?

File: docs/guide/content_rendering/design_engine/add_new_design.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+is the same as in the main site.
+This is because you defined `standard` design as fallback for this SiteAccess:
+
+``` yaml
+[[= include_file('code_samples/front/add_design/config/packages/views.yaml', 0, 3) =]]
+```
+
+In this case, if the design engine cannot find a template for the current design,
+it uses the template from the next configured design.
+In the case above, the engine doesn't find the footer template for the `campaign` SiteAccess,
```
Comment (juskora) [inline docs/guide/content_rendering/design_engine/add_new_design.md:105]:
```suggestion
In the case above, the engine does not find the footer template for the `campaign` SiteAccess,
```

**Review by juskora — CHANGES_REQUESTED** [review]

File: docs/guide/content_rendering/design_engine/add_new_design.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
+# Add new design
+
+To create different designs for different version of the website,
+you configure different sites based on the [SiteAccess](../../multisite/multisite.md) concent.
```
[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/guide/content_rendering/design_engine/add_new_design.md:4]:
```suggestion
you configure different sites based on the [SiteAccess](../../multisite/multisite.md) content.
```
Yep, links are broken until the other PR is merged.

File: docs/guide/content_rendering/design_engine/add_new_design.md (hunk 3 above)
Comment (DominikaK) [reply docs/guide/content_rendering/design_engine/add_new_design.md:31]:
```suggestion
Next, configure a new `summersale` design for this theme, also named `summersale`:
```

**Review by juskora — APPROVED** [review]

**Pull request merged**